oauth2client/contrib/gce.py

```diff
@@ -85,32 +138,128 @@ class AppAssertionCredentials(AssertionCredentials):
     information to generate and refresh its own access tokens.
     """
 
-    @util.positional(2)
-    def __init__(self, scope='', **kwargs):
+    def __init__(self,
+                 scope=None,
+                 service_account_email='default',
```

Comment:
Another option is leave `service_account_email` as `None` and `service_account_info` as `{}`, and change `_refresh` to look like
```
def _refresh(self, http_request):
    self.access_token, self.token_expiry = _get_access_token(
        http_request,
        sef._service_account_info.get('email', 'default')
)
```
Which maybe I like better? Can't decide.